app/forms/bank_details_form.rb

```diff
@@ -26,13 +26,15 @@ class BankDetailsForm < Form
   def save
     return false unless valid?
 
-    update!(
-      banking_name:,
-      bank_sort_code:,
-      bank_account_number:,
-      building_society_roll_number:,
-      hmrc_bank_validation_succeeded:
+    journey_session.answers.assign_attributes(
+      banking_name: banking_name,
```

Comment:
i'm guessing you're not a fan of the implicit syntax 😂

Reply:
No! Every time I look at I think it's a syntax error!